doc/uftrace-live.md

```diff
@@ -48,7 +48,7 @@ COMMON OPTIONS
 
 \--no-libcall
 :   Do not record library function invocations.  Library calls are normally
-    traced by hooking the dynamic linker's resolve function in the PLT.
+    traced by hooking the dynamic linker to resolve function in the PLT.
```

Comment:
Hmm.. I'd rather say "hooking calls to the resolver function of dynamic linker in the PLT".

Reply:
@namhyung Thank you for your opinion. It's more likely natural 👍Fixed it right away.

Reply:
Working done.